.env.sample

```
@@ -9,9 +9,33 @@ BROWSE_INCREMENT="10"
 # "false" to revert filterName to the legacy name-substring regex without a
 # code revert. Any other value (or unset) is enabled.
 # SEARCH_V2_ENABLED="false"
-DB_URI="mongodb://localhost:27017/blueprintnotincluded"
+# Inside the dev container the database and Mailpit are reachable by compose
+# service name, so this is "database" and SMTP_HOST is "mailhog". Running the
+# app straight on the host instead, they are localhost and the published
+# MONGO_PORT / MAILPIT_SMTP_PORT.
+DB_URI="mongodb://database:27017/blueprintnotincluded"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)